README.md

```diff
@@ -36,10 +36,16 @@ directory:
 
     python setup.py install --user --install-scripts ~/bin
 
+For the purposes of testing or development, you may prefer to install PROPKA as
+an editable module via PIP by running
+```
+pip install -e .
+```
+from within a virtual environment (e.g., via [virtualenv](https://pypi.org/project/virtualenv/)).
 
 ## Requirements
 
-* Python 2.7 or higher or Python 3.1 or higher 
+* Python 3.1 or higher 
```

Comment:
Don't know if anyone cares for testing with Python versions lower than 3.4 or even 3.6 these days. I would put at least 3.4 here.

Reply:
Some people in the sciences still use 2.7... but yes, if it's any effort to support 2.7 and <3.4, definitely start at 3.5 or 3.5.

Reply:
I've also had people in the sciences give me a very hard time because I sent them HTML emails that they can't open in PINE or ELM.  I think we should move on to Python 3.*....

Reply:
Addressed by https://github.com/Electrostatics/propka-3.1/commit/48f69eb544c37800e24895d20bbf45a43baa52b5 I think.